content/.prettierrc.js

```diff
@@ -1,5 +0,0 @@
-/* eslint-disable filenames/match-regex */
-module.exports = {
-  parser: 'markdown',
-  proseWrap: 'always',
-};
```

Comment:
Do we still want to use prettier with this config or do we have another formatter for markdown now?

Reply:
I think it would likely make sense to add `prettier` formatting back for markdown. You would also need to add something like this, but for `*.md`:

https://github.com/maxmind/blog-site/blob/45c796b04063f1a18d6ee519f2a09dcfb9c0f2cb/package.json#L180

As well as a dev-dep on `prettier`.

Reply:
:+1: Maybe we should add a linter check for `prettier` in GitHub Actions. Given the number of things that changed, it is pretty clear we were not running it regularly previously.